bin/ci.branches

```diff
@@ -9,5 +9,4 @@ git submodule update
 
 ./cabal update
 ./cabal build
-./cabal test --log=/dev/stdout
-
+./cabal testci --log=/dev/null
```

Comment:
Not sure what was the story behind this, from what I can tell the log is always the stdout from the tests anyway, so this just causes duplicate test output to be written in Jenkins.